README.md

```diff
@@ -59,9 +59,19 @@ After running the installation script, NErlNet directory can be accessed via the
 ## Python API and Jupyter: 
 
 1. Create and activate a virtual environment for Nerlnet https://docs.python.org/3/library/venv.html
-2. Install required modules ```pip install -r src_py/requirements.txt```
+2. Install required modules ```pip install -r src_py/reqCleaned.txt```
```

Comment:
@halfway258 
Don't change the name of requirements file 
requirements.txt name is counted as a standard

Reply:
Fixed